### PR TITLE
UI: Launched jobs quality of life improvements

### PR DESCRIPTION
**Pull request by DingoEatingFuzz**
This adds three things to improve the job detail pages for periodic and parameterized jobs.

1. Adds the submitted time as a column in the table of job launches.
2. Default sorts by most recent submission.
3. Adds the same page size selector and global page size setting used elsewhere in the UI.

image

### CODE REVIEW

File: ui/app/components/job-row.js [hunk 1]
```diff
@@ -12,6 +12,10 @@ export default class JobRow extends Component {
 
   job = null;
 
+  // One of independent, parent, or child. Used to customize the template
+  // based on the relationship of this job to others.
+  context = 'independent';
```
Comment (backspace) [inline ui/app/components/job-row.js:17]: I couldn’t find a use of `parent` for this, did you change your mind on that?

**Review by backspace — APPROVED** [review]
This looks like a nice improvement 😀 I had one minor observation about a maybe-outdated comment but it’s non-blocking.

File: ui/app/components/job-row.js (hunk 1 above)
Comment (DingoEatingFuzz) [reply ui/app/components/job-row.js:17]: This was a note for future us for where I was going with this. These are the three types of contexts I was imagining when I stuck this field in here.

The fact it needs an explanation may mean the attr name isn't good. But it's also consistent with AllocationRow.

Comment (backspace) [reply ui/app/components/job-row.js:17]: oh yes, makes sense. Well I don’t have a better name suggestion and the comment helps contextualise 😆

Comment (DingoEatingFuzz) [reply ui/app/components/job-row.js:17]: I'm gonna merge it then 🙂

**Pull request merged**